nec_tables.py

```diff
@@ -460,178 +460,434 @@
+                         '500': 0.048,
+                         '600': 0.048,
+                         '750': 0.048,
+                         '1000': 0.046}}
+
+# Copper conductor resistance (R) per 1000 FEET from NEC 2017 Chapter 9, Table 9
+# XX:{YY: ZZ}  :: XX=conduit_material, YY=cond_size, ZZ=cond_reactance
+cond_resistance_cu =  {'PVC':{
+                         '12': 2.000,
+                         '10': 1.200,
+                         '8': 0.780,
+                         '6': 0.490,
+                         '4': 0.310,
+                         '3': 0.250,
+                         '2': 0.190,
+                         '1': 0.150,
+                         '1/0': 0.120,
+                         '2/0': 0.100,
+                         '3/0': 0.077,
+                         '4/0': 0.062,
+                         '250': 0.052,
+                         '300': 0.044,
+                         '350': 0.038,
+                         '400': 0.033,
+                         '500': 0.027,
+                         '600': 0.023,
+                         '750': 0.019,
+                         '1000': 0.015},
+                   'Alum':{
+                         '12': 2.000,
+                         '10': 1.200,
+                         '8': 0.780,
+                         '6': 0.490,
+                         '4': 0.310,
+                         '3': 0.250,
+                         '2': 0.200,
+                         '1': 0.160,
+                         '1/0': 0.130,
+                         '2/0': 0.100,
+                         '3/0': 0.082,
+                         '4/0': 0.067,
+                         '250': 0.057,
+                         '300': 0.049,
+                         '350': 0.043,
+                         '400': 0.038,
+                         '500': 0.032,
+                         '600': 0.028,
+                         '750': 0.024,
+                         '1000': 0.019},
+                   'Steel':{
+                         '12': 2.000,
+                         '10': 1.200,
+                         '8': 0.780,
+                         '6': 0.490,
+                         '4': 0.310,
+                         '3': 0.250,
+                         '2': 0.200,
+                         '1': 0.160,
+                         '1/0': 0.120,
+                         '2/0': 0.100,
+                         '3/0': 0.079,
+                         '4/0': 0.063,
+                         '250': 0.054,
+                         '300': 0.045,
+                         '350': 0.039,
+                         '400': 0.035,
+                         '500': 0.029,
+                         '600': 0.025,
+                         '750': 0.021,
+                         '1000': 0.018}}
+
+# Aluminum conductor resistance (R) per 1000 FEET from NEC 2017 Chapter 9, Table 9
+# XX:{YY: ZZ}  :: XX=conduit_material, YY=cond_size, ZZ=cond_reactance
+cond_resistance_al =  {'PVC':{
```

Comment:
Should the aluminum abbreviation be consistent? The abbreviation used for the key in the conductor resistance dictionary is 'Alum'.  Should this be `cond_resistance_alum`?

Reply:
Changed to alum